sentry_streams/sentry_streams/adapters/stream_adapter.py

```diff
@@ -96,6 +98,28 @@ def reduce(
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def router(
+        self,
+        step: Router,
+        stream: Stream,
+    ) -> Stream:
+        """
+        Build a router operator for the platform the adapter supports.
+        """
+        raise NotImplementedError
+
+    @abstractmethod
+    def branch(
+        self,
+        step: Branch,
+        stream: Stream,
+    ) -> Stream:
+        """
+        Branch represents a possible branch downstream from a router.
+        """
+        raise NotImplementedError
```

Comment:
You do not need the branch form the adapter. This function will not be called. While you need a branch step as a place holder in the `step_streams` object to wire branch steps onto a stream, you do not need to actually translate the Branch step object. That is performed when translating the Router.